### PR TITLE
Added drag and drop functions, did not yet add the update event

### PR DESCRIPTION
**Pull request by doyouthunderstand**
Also made pieces into just pieces and not links, therefore removing access to the pieces show page. have not yet removed the pieces show page however.

### CODE REVIEW

File: app/views/games/show.html.erb [hunk 1]
```diff
@@ -52,7 +59,32 @@
...
+      $('.tile').droppable( {
+        drop: updatePieceLocation
+      });
+      
+    });
+   
+    var current_piece = ui.draggable.data('current-piece');
+    
+    // I honestly know almost nothing about ajax calls so this could be wrong as well
+    function updatePieceLocation() {
```
Comment (mscoutermarsh) [inline app/views/games/show.html.erb]: change this to `function updatePieceLocation(event, ui) {`

File: app/views/games/show.html.erb [hunk 2]
```diff
@@ -52,7 +59,32 @@
...
 </div>
 
-
+<script>
+    $(function(){
+      
+      $('.piece').draggable();
+      
+      $('.tile').droppable( {
+        drop: updatePieceLocation
```
Comment (mscoutermarsh) [inline app/views/games/show.html.erb]: change this to `drop: function(event, ui) { updatePieceLocation(event, ui); }`

**Comment by doyouthunderstand** [comment]
@mscoutermarsh just took a look at those changes, could you explain event.target.dataset[]?

**Comment by mscoutermarsh** [comment]
@doyouthunderstand We need a way to get the x/y coordinate of the tile we are dropping on.

For some reason (I'm not sure). `ui.droppable` was not returning the tile.

`event.target` was returning the correct tile.

You can get the data attributes by using `event.target.dataSet`

**Pull request merged**